language/move-prover/tests/sources/functional/bitwise_table.move

```diff
@@ -0,0 +1,130 @@
+module 0x42::VerifyBitwiseTable {
+    use extensions::table::{Self, Table};
+    use extensions::table::{spec_get, spec_len, spec_contains};
+
+    fun add(): Table<u8, u64> {
+        let t = table::new<u8, u64>();
+        table::add(&mut t, 1, 2);
+        table::add(&mut t, 2, 3);
+        table::add(&mut t, 3, 4);
+        t
+    }
+    spec add {
+        pragma bv_ret=b"0";
```

Comment:
This means the value part of `Table<u8, u64>` (which is `u64`) is to be encoded as `bv`? What happens for `Table<u8, Table<u8, u64>>`? (I am not sure if this type can practically exist though...)

Reply:
To make it work, values in vectors and tables need to be integer types.

Reply:
What happens if not? Does it also give an internal boogie error?

Reply:
Yes, I added it to the doc.